### DISCUSSION TITLE
Keycloak 17.x Quarkus Random High Port

### DISCUSSION THREAD

**Discussion by XobrekX** [keycloak/keycloak#13099] [Q&A]
Hello everybody,

I am responsible for a keycloak cluster and do monitor the whole systems and all parts of it. 
With the latest update of keycloak, there appeared a new local port, I was not able to configure. 
For me, keycloak opens 4 ports, three of them, I was able to set via jgroups

TCP 7600:
`
<TCP bind_port="7600"
         sock_conn_timeout="2000"
         send_buf_size="640000"
         enable_diagnostics="false"
         />
`
TCP 57600:
`<FD_SOCK start_port="57600"/>`

TCP 8505 is the http port.

So there is one port left, a local high port. Usually it is over 30000. I was not able to figure out what this port is used for and how to configure it to a static value

```
tcp     LISTEN   0        50           10.19.68.14:7600           0.0.0.0:*      users:(("java",pid=773545,fd=493))
tcp     LISTEN   0        50             127.0.0.1:8505           0.0.0.0:*      users:(("java",pid=773545,fd=499))
tcp     LISTEN   0        50             127.0.0.1:40349          0.0.0.0:*      users:(("java",pid=773545,fd=498))
tcp     LISTEN   0        50           10.19.68.14:57600          0.0.0.0:*      users:(("java",pid=773545,fd=494))
```

Hope you can help me and give clearity about this port. Thank you!

**Comment by XobrekX** [comment]
Anyone an idea?

**Reply by olivierdepriester** [reply]
I figured out I have also 1 unknown listening port, even in a standalone configuration. I am curious about its usage and how it is aimed to be reached. I telnet some stuff on it but it does not seem to be consumed.
```
tcp        0      0 0.0.0.0:8085            0.0.0.0:*               LISTEN      1000       1565619    53401/java          
tcp        1      0 127.0.0.1:32921         0.0.0.0:*               LISTEN      1000       1545637    53401/java          
tcp        0      0 127.0.0.1:8787          0.0.0.0:*               LISTEN      1000       1565564    53401/java
```

**Comment by lrozenblyum** [comment]
Same for Keycloak 22.0.5
